main.go

```diff
@@ -102,6 +115,23 @@ func main() {
 	}
 }
 
+func snapshotProfiles(dir string) (func() error, error) {
+	// TODO: now -> date
+	if err := os.MkdirAll(filepath.Join(dir, "now"), os.ModePerm); err != nil {
+		return nil, err
+	}
+	f, err := os.OpenFile(filepath.Join(dir, "now", "fgprof.pprof"), os.O_TRUNC|os.O_WRONLY, os.ModePerm)
```

Comment:
```suggestion
	f, err := os.OpenFile(filepath.Join(dir, "now", "fgprof.pprof"), os.O_CREATE|os.O_TRUNC|os.O_WRONLY, os.ModePerm)
```
This would create a new file if none exists, which is why there was error before. 🙂

Reply:
![Screenshot 2021-06-25 at 10 49 36 AM](https://user-images.githubusercontent.com/51132453/123373469-12feb500-d5a3-11eb-8460-67160f16b6f8.png)
Works with Thanos now! 🙂